src/server/Main.cpp

```diff
@@ -60,19 +42,27 @@ void displayPolymorphic(Engine::GameEngine &engine, It begin, It end)
             for (const auto &byte : comp->serialize()) {
                 if (i != 0)
                     std::cout << ", ";
-                std::cout << (int)byte;
+                std::cout << (unsigned)byte;
                 i++;
             }
             std::cout << "}" << std::endl;
         }
+        std::cout << std::endl;
     }
 }
 
 void updateComponent(size_t id, std::string name, std::vector<uint8_t> data)
 {
     std::cout << "Updating component: " << name << " with ID: " << id << std::endl;
-    for (auto u : data)
-        std::cout << u << " ";
+    std::cout << "    {";
+    int i = 0;
+    for (const auto &byte : data) {
+        if (i != 0)
+            std::cout << ", ";
+        std::cout << (unsigned)byte;
+        i++;
+    }
+    std::cout << "}" << std::endl;
 }
```

Comment:
what is this function used for ?

Reply:
Just debug right now, but it will be the network call back for sending component updates to the client later on.

Reply:
okay, that is what I thought but since it wasn't doing anything right now I was wondering what this could be